content/projects/cptox21.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Cptox21"
```

Comment:
Why do we still see this page under "preview all projects" even though @AndreaVolkamer said that this project will only be launched later?

Reply:
I guess that happened by mistake, @jaimergp I can't find where to exclude it, could you take over here?

Reply:
Fixed!

Reply:
so `draft: true` does the trick :)